piet-web/examples/basic/Cargo.toml

```diff
@@ -7,12 +7,17 @@ edition = "2018"
 [lib]
 crate-type = ["cdylib"]
 
+[features]
+default = ["console_error_panic_hook"]
+
 [dependencies]
 piet = { path = "../../../piet" }
 piet-web = { path = "../.." }
 piet-test = { path = "../../../piet-test" }
 
 wasm-bindgen = "0.2.30"
+console_error_panic_hook = { version = "0.1.1", optional = true }
```

Comment:
Sorry I didn't notice this earlier, but is there a specific reason this is `0.1.1`? `0.1.6` seems to be the latest.

Reply:
No reason, just copied from the [rust game of life tutorial](https://github.com/rustwasm/wasm_game_of_life/blob/master/Cargo.toml)